metadata-db-app/src/cmr/metadata_db/data/oracle/sql_helper.clj

```diff
@@ -5,10 +5,22 @@
    [clojure.java.jdbc :as j]
    [clojure.string :as str]
    [cmr.common.services.errors :as errors]
+   [cmr.common.log :refer [debug error info trace warn]]
    [cmr.metadata-db.data.oracle.concept-tables :as ct]
-   [cmr.oracle.sql-utils :as su :refer [insert values select from where with order-by desc delete as]])
+   [cmr.oracle.sql-utils :as su :refer [insert values select from where with order-by desc delete as]]
+   [cmr.efs.config :as efs-config]
+   [cmr.efs.connection :as efs]
+   [cmr.common.util :as util])
   (:import cmr.oracle.connection.OracleStore))
 
+;;----------- HELPER FOR EFS DELETE --------------
```

Comment:
nit; maybe this function could just have a doc-string that explains why it's needed instead of a comment up here